backend/src/infra/controller/GroupController.ts

```diff
@@ -65,4 +67,27 @@ export const GroupController: FastifyPluginAsync<{
       return result.payload;
     }
   );
+
+  const calculBalance = new CalculGroupBalanceUseCase(group);
+  fastify.post<{ Body: RenameGroupInput; Params: GroupId }>(
+    "/:id/calculBalance",
```

Comment:
Should be `GET /:id/balance`, because it doesn't change anything, it just computes and returned the group balance